api-report/map.api.md

```diff
@@ -50,6 +50,11 @@ export interface IDirectory extends Map<string, any>, IEventProvider<IDirectoryE
     subdirectories(): IterableIterator<[string, IDirectory]>;
 }
 
+// @public
+export interface IDirectoryCreated {
```

Comment:
If it's only one property, then I'd think we do not need interface - argument can be a string.

Reply:
Since the ISubDirectoryCreated extends this, we need to have the interface,

Reply:
Replaced it with reltive path